### PR TITLE
Base calculator

### PR DESCRIPTION
**Pull request by alissonperes**
Create react app
Deploy to heroku

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,46 @@
+{
+  "name": "mv_simple_react_calculator",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^4.2.4",
+    "@testing-library/react": "^9.3.2",
+    "@testing-library/user-event": "^7.1.2",
+    "prop-types": "^15.7.2",
```
Comment (bellom) [inline package.json:9]: We need to add `big.js`, seems you miss that 😄 
image

File: README.md [hunk 2]
```diff
@@ -1 +1,42 @@
-# mv_simple_react_calculator
+# Building a simple calculator APP with React
+
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+
```
Comment (bellom) [inline README.md:4]: It would be nice to add the `Heroku live link` in here 👍

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi @alissonperes :clap: 

### Awesome job so far!! :+1: 

#### You have done wonderfully well, you passed `almost` all the checklist for this project :+1: just a few issues left to work on............ :slightly_smiling_face: 
 
**Kindly check out my comments below;**

**Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....**

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

File: package.json (hunk 1 above)
Comment (alissonperes) [reply package.json:9]: I've aadded it in devDependencies. Thanks for the heads up, I'm changing it.

File: README.md (hunk 2 above)
Comment (alissonperes) [reply README.md:4]: Added live version link to readme as requested.

**Review by bellom — APPROVED** [review]
## Hi @alissonperes :clap: 

**Great work making all the necessary corrections from the previous review.**

## Congratulation :+1:  
_Go ahead and merge to dev and don't forget to submit project completion form.
Good luck in the next milestone :slightly_smiling_face:_ 

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Pull request merged**